src/Analyzer.JsonRuleEngine/Expressions/StructuredExpression.cs

```diff
@@ -32,25 +33,27 @@ public StructuredExpression(Expression[] expressions, Func<bool,bool, bool> oper
             : base(commonProperties)
         {
             Expressions = expressions ?? throw new ArgumentNullException(nameof(expressions));
-            Operation = operation;
+            Operation = operation ?? throw new ArgumentNullException(nameof(operation));
         }
 
         /// <summary>
         /// Evaluates all expressions provided and aggregates them in a final <see cref="JsonRuleEvaluation"/>
         /// </summary>
         /// <param name="jsonScope">The json to evaluate.</param>
+        /// <param name="jsonLineNumberResolver">An <see cref="ILineNumberResolver"/> to
```

Comment:
For these can't we just use the inherit doc tag?

Reply:
It looks like the documentation enforcement mechanism we're using doesn't work well with `<inheritdoc/>`.  Several of these properties have different wording than in `Expression`, so the documentation is redone here.

While documentation seems to work when specifying `<inheritdoc/>` *and* overriding some elements, if I try to override one parameter but not another, I get a build error for the parameter not overridden.